lib-multisrc/comicaso/src/eu/kanade/tachiyomi/multisrc/comicaso/Comicaso.kt

```diff
@@ -0,0 +1,502 @@
+            "Royalty" -> "royalty"
+            "School Life" -> "school-life"
+            "Sci-Fi", "Sci Fi" -> "sci-fi"
+            "Seinen(M)", "Seinenm" -> "seinenm"
+            "Sejarah" -> "sejarah"
+            "Shoujo(G)", "Shoujog" -> "shoujog"
+            "Shoujo ai", "Shoujo Ai" -> "shoujo-ai"
+            "Shounen ai", "Shounen Ai" -> "shounen-ai"
+            "Shounen(B)", "Shounenb" -> "shounenb"
+            "Showbiz" -> "showbiz"
+            "Slice of Life", "Slice Of Life" -> "slice-of-life"
+            "SM/BDSM/SUB-DOM", "SM Bdsm Sub Dom" -> "sm-bdsm-sub-dom"
+            "Smut" -> "smut"
+            "Space" -> "space"
+            "Sports" -> "sports"
+            "Super Power" -> "super-power"
+            "Superhero" -> "superhero"
+            "Supernatural" -> "supernatural"
+            "Survival" -> "survival"
+            "Thriller" -> "thriller"
+            "Time Travel" -> "time-travel"
+            "Tower Climbing" -> "tower-climbing"
+            "Tragedy" -> "tragedy"
+            "Transmigration" -> "transmigration"
+            "Vampires" -> "vampires"
+            "Video Games" -> "video-games"
+            "Villainess" -> "villainess"
+            "Violence" -> "violence"
+            "Western" -> "western"
+            "Wuxia" -> "wuxia"
+            "Yakuzas" -> "yakuzas"
+            "Yaoi(BL)", "Yaoibl" -> "yaoibl"
+            "Yuri(GL)", "Yurigl" -> "yurigl"
+            "Zombies" -> "zombies"
+            else -> genre.lowercase()
+        }
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val json = JSONObject(response.body.string())
+        val items = json.getJSONArray("items")
+        return MangasPage(parseMangaList(items), items.length() == pageSize)
+    }
+
+    private fun parseMangaList(json: JSONArray): List<SManga> {
+        val list = ArrayList<SManga>(json.length())
+        for (i in 0 until json.length()) {
+            val obj = json.getJSONObject(i)
+            val slug = obj.getString("slug")
+            val url = "/v2/manga/$slug/"
+            list.add(
+                SManga.create().apply {
+                    this.url = url
+                    title = obj.getString("title")
+                    thumbnail_url = obj.getString("thumb")
+                    status = when (obj.optString("status")) {
+                        "on-going" -> SManga.ONGOING
+                        "completed" -> SManga.COMPLETED
+                        else -> SManga.UNKNOWN
+                    }
+                },
+            )
+        }
+        return list
+    }
+
+    override fun mangaDetailsParse(document: Document) = SManga.create().apply {
+        title = document.selectFirst("h1.ng-detail-title")!!.text().trim()
+
+        description = document.selectFirst("p.ng-desc")?.text()?.trim()
+
+        val altTitlesText = document.selectFirst(".ng-meta-info p:contains(Alternative:)")
+            ?.ownText()
+            ?.trim()
```

Comment:
`.ownText` is also trimmed.
```suggestion
            ?.ownText()
```

Suggestion applies throughout this file.